opentech/apply/projects/templates/application_projects/includes/payment_requests.html

```diff
@@ -10,44 +15,45 @@
+                <th class="payment-block__table-update">Change status</th>
             </tr>
         </thead>
         <tbody>
+            {% for payment_request in payments.not_rejected %}
             <tr>
-                <td><span class="payment-block__mobile-label">Amount: </span>$10,000</td>
+                <td><span class="payment-block__mobile-label">Amount: </span>${{ payment_request.value }}</td>
                 <td><span class="payment-block__mobile-label">Status: </span>Approved 1111-11-11</td>
                 <td><span class="payment-block__mobile-label">Documents: </span><a href="#">Download</a></td>
                 <td>
                     <span class="payment-block__mobile-label">Status:</span>
-                    <span class="payment-block__status">Approved</span>
-                    <a data-fancybox data-src="#change-payment-status" class="payment-block__status-link" href="#">Change status</a>
-                </td>
-            </tr>
-            <tr>
-                <td><span class="payment-block__mobile-label">Amount: </span>$1,000</td>
-                <td><span class="payment-block__mobile-label">Status: </span>Approved 2222-11-11</td>
-                <td><span class="payment-block__mobile-label">Documents: </span><a href="#">Download</a></td>
-                <td>
-                    <span class="payment-block__mobile-label">Status:</span>
-                    <span class="payment-block__status">Paid</span>
-                    <a data-fancybox data-src="#change-payment-status" class="payment-block__status-link" href="#">Change status</a>
+                    {% comment %}
+                    TODO: front end fix to make these dropdowns
+                    {% for status in payments.availabe_statuses %}
+                    <span class="payment-block__status">{{ status.0 }}{{ status.1 }}</span>
+                    {% endfor %}
+                    {% endcomment %}
+                    <a data-fancybox data-src="#change-payment-status-{{ payment_request.pk }}" class="payment-block__status-link" href="#">Change status</a>
                 </td>
             </tr>
+            {% endfor %}
         </tbody>
     </table>
-    <p class="payment-block__rejected"><a class="payment-block__rejected-link js-payment-block-rejected-link" href="#">Show rejected</a></p>
+
+    <p class="payment-block__rejected">
+        <a class="payment-block__rejected-link js-payment-block-rejected-link" href="#">Show rejected</a>
+    </p>
 
     <table class="payment-block__table is-hidden js-payment-block-rejected-table">
         <thead>
             <tr>
                 <th class="payment-block__table-amount">Amount</th>
                 <th class="payment-block__table-status">Status</th>
                 <th class="payment-block__table-docs">Documents</th>
-                <th class="payment-block__table-update">Status</th>
+                <th class="payment-block__table-update">Change status</th>
```

Comment:
We didn't have a design for the rejected table, but i don't think it makes sense to include the status change column.